### PR TITLE
Create utility class for QR Code FX image creation

### PR DESCRIPTION
**Pull request by msgilligan**
Move the code to generate a JavaFX image for a QR code to a reusable
utility method.

### CODE REVIEW

File: wallettemplate/src/main/java/wallettemplate/utils/QRCodeImages.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
+package wallettemplate.utils;
```
Comment (schildbach) [inline wallettemplate/src/main/java/wallettemplate/utils/QRCodeImages.java]: Can you add the license header?

Comment (msgilligan) [reply wallettemplate/src/main/java/wallettemplate/utils/QRCodeImages.java]: Added!

**Comment by msgilligan** [comment]
@schildbach The build failure looks like it will clear if the job is resubmitted.

**Comment by schildbach** [comment]
Merged!

**Pull request closed without merging**